tests/system/python/conftest.py

```diff
@@ -951,3 +978,34 @@ def pytest_itemcollected(item):
     suf = node.__doc__.strip() if node.__doc__ else node.__name__
     if pref or suf:
         item._nodeid = ' '.join((pref, suf))
+
+
+# Parameters required for testing Fledge under an impaired or noisy network.
+@pytest.fixture
+def south_service_wait_time(request):
+    return request.config.getoption("--south-service-wait-time")
+
+
+@pytest.fixture
+def north_catch_up_time(request):
+    return request.config.getoption("--north-catch-up-time")
+
+
+@pytest.fixture
+def interface_for_impairment(request):
+    return request.config.getoption("--interface-for-impairment")
+
+
+@pytest.fixture
+def rate_limit(request):
+    return request.config.getoption("--rate-limit")
+
+
+@pytest.fixture
+def packet_delay(request):
+    return request.config.getoption("--packet-delay")
+
+
+@pytest.fixture
+def start_north_as_service(request):
```

Comment:
not needed as a fixture, we can pass local scope var to call corresponding fixture

Reply:
But why not keep it configurable.